transcom/mymove
# B 17849 restrict records returned by prime simulator

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A864282&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Prime Simulator is pulling too many records and is bogged down. Added filter on API call to pull only recent moves.
Urgent change requested by Dan Schuster

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a prime
3. Page should now load only 20 records at a time

### Frontend


### Backend
